Makefile

```diff
@@ -90,7 +90,7 @@ examples:
 # This is largely libtock_runtime and crates that depend on libtock_runtime.
 # Used when we need to build a crate for the host OS, as libtock_runtime only
 # supports running on Tock.
-EXCLUDE_RUNTIME := --exclude libtock2 --exclude libtock_runtime --exclude libtock_small_panic
+EXCLUDE_RUNTIME := --exclude libtock2 --exclude libtock_runtime --exclude libtock_console_panic --exclude libtock_small_panic
```

Comment:
Please line-wrap this line (similar to how `EXCLUDE_STD`'s definition is line-wrapped).

Reply:
Done - I eschewed the wrapping of EXCLUDE_STD because it's spaces while everything else uses tabs.